modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/edit_file_entry_picker.jspf

```diff
@@ -68,7 +68,15 @@ DLFilePicker dlFilePicker = dlEditFileEntryDisplayContext.getDLFilePicker(lifera
 				var titleElement = document.getElementById('<portlet:namespace />title');
 
 				if (titleElement) {
-					titleElement.value = documentTitle;
+					titleElement.value = documentTitle.replace(/\.[^.]*$/, '');
```

Comment:
In my view using `documentName` instead of `documentTitle` seems more understandable.

Reply:
Great suggestion!
I will change it!!

Thanks!